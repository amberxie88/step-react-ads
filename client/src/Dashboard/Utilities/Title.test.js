// Link.react.test.js
import React from 'react';
import Title from './Title';
import renderer from 'react-test-renderer';

test('Title renders correct elements', () => {
  const component = renderer.create(<Title>Sample Text</Title>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
