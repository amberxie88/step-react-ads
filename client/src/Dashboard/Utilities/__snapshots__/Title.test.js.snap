// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correct elements 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
>
  Sample Text
</h2>
`;
